index.js

```diff
@@ -61,7 +63,7 @@ module.exports = app => {
           name: releaseInfo.name,
           tag_name: releaseInfo.tag,
           body: releaseInfo.body,
-          draft: true,
+          draft: !shouldPublish,

```

Comment:
Not a understandable name in context of draft especially because you negate the condition.
`shouldDraft` might make more sense and then you would not have to negate here?

Reply:
Sure, ok. I’m happy to make these changes, but I’d also love to get a sense of whether the overall approach here is something likely to be accepted or if that would be wasted effort 🙂